test/test_utils/sagemaker.py

```diff
@@ -90,7 +90,8 @@ def generate_sagemaker_pytest_cmd(image, sagemaker_test_type):
     :param sagemaker_test_type: local or remote test type
     :return: <tuple> pytest command to be run, path where it should be executed, image tag
     """
-    reruns = 4
+    reruns = 1
```

Comment:
would it run test just once?

Reply:
yes, it would `rerun` the failed test once.